src/main/java/camp/woowak/lab/coupon/domain/CouponIssuance.java

```diff
@@ -0,0 +1,51 @@
+package camp.woowak.lab.coupon.domain;
+
+import java.time.LocalDateTime;
+
+import org.springframework.data.annotation.CreatedDate;
+
+import camp.woowak.lab.customer.domain.Customer;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import lombok.Getter;
+
+@Entity
+@Getter
+public class CouponIssuance {
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
+
+	@JoinColumn(name = "coupon_id", nullable = false)
+	@ManyToOne(fetch = FetchType.LAZY)
+	private Coupon coupon;
+
+	@JoinColumn(name = "customer_id", nullable = false)
+	@ManyToOne(fetch = FetchType.LAZY)
+	private Customer customer;
+
+	@Column(nullable = false)
```

Comment:
`issuedAt` 과 같이 생성된 시간에 대한 컬럼은 updatable을 false로 설정해서 변경되지 않도록 강제해주는게 좋을것 같아요!
```suggestion
	@Column(nullable = false, updatable = false)
```